pom.xml

```diff
@@ -215,6 +215,9 @@
 				<configuration>
 					<!-- The package of your generated sources -->
 					<packageName>tds.exam.results.trt</packageName>
+                    <sources>
+                        <source>src/main/resources/xsd</source>
```

Comment:
We can add other sources if we want or need to load XSDs that we don't want on the classpath.